scholia/app/templates/taxon_metabolome.sparql

```diff
@@ -14,7 +14,7 @@ WITH {
     UNION
     { ?metabolite wdt:P1582 ?children }
     VALUES ?chemical { wd:Q11173 wd:Q59199015 wd:Q56256086 wd:Q15711994 }.
-    ?metabolite wdt:P31|wdt:P279/wdt:P31 ?chemical .
+    ?metabolite wdt:P31|wdt:P279/wdt:P31|wdt:P279 ?chemical .
```

Comment:
Would replace by `?metabolite wdt:P31 ?chemical .`

Reply:
Thanks ! Took into account in [359f2e8](https://github.com/WDscholia/scholia/pull/2334/commits/359f2e8130a8bda02860e757d22c6dc603b12e42)

Reply:
🚀